src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Types/Type.cs

```diff
@@ -585,16 +585,6 @@ public PredefinedType getPredefType()
             return getAggregate().GetPredefType();
         }
 
-        ////////////////////////////////////////////////////////////////////////////////
-        // Is this type System.TypedReference or System.ArgIterator?
-        // (used for errors because these types can't go certain places)
-
-        public bool isSpecialByRefType()
-        {
-            // ArgIterator, TypedReference and RuntimeArgumentHandle are not supported.
```

Comment:
TypedReference and ArgIterator are back in NetStandard (and AFAIK are not going away), so maybe the right fix would be to restore the checks for those in isSpecialByRefType. This should eventually be replaced by whatever API gets approved for #17232.